### ISSUE TITLE
Travis Cl and Appveyor CI failing

### ISSUE THREAD

**Issue by santoshphilip** [santoshphilip/eppy#170]
I made a branch `i170_develop` from branch `develop` to illustrate the problem.
When branch `i170_develop` is pushed to GitHub, Travis Cl give the following message:
<https://travis-ci.org/santoshphilip/eppy/builds/280992265?utm_source=email&utm_medium=notification> indicating the failure

This used to pass earlier in Travis Cl. I think Travis has made some change in their setup, which is triggering the error. I am unable to recreate the error on my development machine (a Mac)

@jamiebull1 can you take a look into this when you get a chance ? You understand this better than I do.

**Comment by jamiebull1** [comment]
Sure, I'll take a look now

**Comment by jamiebull1** [comment]
[349.2](https://travis-ci.org/santoshphilip/eppy/jobs/280992268) looks like an intermittent error that we see sometimes when the EnergyPlus dowenload fails for whatever reason.

The other two are failing because there's some problem installing (or finding) `pydot3k` on the linux build. It's not even trying to find it. Not sure if it's related but it looks like Travis is using Python 3.4 although we request 3.5.

There's a similar issue on appveyor, which looks to be due to missing `libxml2` and `libxslt`.

**Comment by santoshphilip** [comment]
we can live with 349.2 error

Do you see a way around the `pydot3k` issue on linux. Can we open an issue with Travis Cl? 
Even if we go that route, I think we are unlikely to get support, since we are using a free service.

**Comment by jamiebull1** [comment]
I'm fairly sure if you rerun it, the 349.2 one will go away.

On the `pydot3k`, I'm not 100% sure what the issue is and I don't have linux machine with me to see if I can reproduce it outside of Travis (I'm in the Czech Republic for a couple of weeks). If I get time to set up an AWS instance I'll try it that way but not sure when that will be.

**Comment by santoshphilip** [comment]
Enjoy the Czech republic. Hope it is a vacation.

I'll putz with this a bit, either on a local VM or on AWS and see what comes up.

**Comment by jamiebull1** [comment]
Thanks. Mainly work (new job), with a bit of play when I can fit it in.

Update here if you find anything so if I get time I can follow up.

**Comment by santoshphilip** [comment]
I did some quick tests on a local virtual machine with Ubuntu. Some observations:
- it had an old version of setuptools
- pydot3k gave an error while attempting to install
- once setuptools was upgraded, pydot3k installed

**Comment by jamiebull1** [comment]
Cool, good find. I'll update the CI script tomorrow and see if that fixes it.

**Comment by jamiebull1** [comment]
This is fixed. The issue was where the python version was being downgraded to 3.4 by Travis, while we were only installing pydot3k for versions >=  3.5. I've changed that to >= 3.0 and it's fixed the issue.

We're still having the issue with Appveyor though. I've got it fixed on Python 2.7 by using Miniconda to handle the libxml dependencies, but getting issues with Miniconda3 for Python 3.5. I've raised an issue [here](https://github.com/conda/conda/issues/6266).
